optimade/filtertransformers/base_transformer.py

```diff
@@ -12,7 +57,7 @@ class BaseTransformer(abc.ABC, Transformer):
 
     """
 
-    # map from standard comparison operators to the backend-specific version
+    """A map from comparison operators to their backend-specific versions"""  # pylint: disable=pointless-string-statement
```

Comment:
I guess not! I'd tricked myself into thinking this would be added as attribute level docs, but I think that is only the case if there is no top-level class docstring. I will tweak this and make sure that `mapper` has some attribute-level docs too

Reply:
Fixed in d7a435c - we now have quite a lot of duplication between attribute and __init__ parameter docstrings. Is there a more elegant solution?